libs/langgraph/src/pregel/index.ts

```diff
@@ -775,7 +875,12 @@ export class Pregel<
       await runManager?.handleChainError(e);
       throw e;
     } finally {
-      await Promise.all(loop?.checkpointerPromises ?? []);
+      this.store?.stop();
+      loop?.store?.stop();
```

Comment:
Why would there be two different values here?

Reply:
fixed